libcaf_test/caf/test/test.test.cpp

```diff
@@ -7,33 +7,37 @@
 
 using caf::test::block_type;
 
-TEST("tests can contain different type of checks") {
+TEST("tests can contain different types of checks") {
   auto* rep = caf::test::reporter::instance;
-  /// tests all of the possible scenario for check_ne
-  check_ne(0u, 0u);
-  check_ne(0u, 1u);
   auto stats = rep->test_stats();
-  /// tests all of the possible scenario for check_eq
-  check_eq(stats.passed, 1u);
-  check_eq(stats.passed, 0u);
-  /// tests all of the possible scenario for check_ge
-  check_ge(stats.passed, 0u);
-  check_ge(stats.passed, 1u);
-  check_ge(stats.passed, 2u);
-  /// tests all of the possible scenario for check_gt
-  check_gt(stats.passed, 0u);
-  check_gt(stats.passed, 1u);
-  check_gt(stats.passed, 2u);
-  /// tests all of the possible scenario for check_le
-  check_le(stats.passed, 0u);
-  check_le(stats.passed, 1u);
-  check_le(stats.passed, 2u);
-  /// tests all of the possible scenario for check_lt
-  check_lt(stats.passed, 0u);
-  check_lt(stats.passed, 1u);
-  check_lt(stats.passed, 2u);
-  info("reset error count to not fail the test");
-  caf::test::reporter::instance->test_stats({16, 0});
+  SECTION("checks functionality of check_ne") {
```

Comment:
Tests and sections should have descriptive titles, not a description of what the section does (applies to all section titles). For example:

```suggestion
  SECTION("check_ne checks for inequality") {
```